v0.27: Queries limited to 1000 returned results (#1607)
meilisearch/documentation: reference/api/search.md

Search for documents matching a specific query in the given index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.

> :::note

Reviewer: ```suggestion
::: note
```

> This route has a [non-customizable limit of 1000 results](/learn/advanced/known_limitations.md#maximum-number-of-results-per-search). If you want to scrap your database, you can use the [get documents endpoint](/reference/api/documents.md#get-documents) instead.

Author: `If you want to scrap your database, you can use the [get documents endpoint](/reference/api/documents.md#get-documents) instead.`
Should we mention this here? Or is the link to known limitations enough?

Reviewer: I like it! Seems like a reasonably helpful tip.

> This endpoint has a [non-customizable limit of 1000 results](/learn/advanced/known_limitations.md#maximum-number-of-results-per-search). If you want to scrap your database, you can use the [get documents endpoint](/reference/api/documents.md#get-documents) instead.

Reviewer: ```suggestion
This endpoint has a [non-customizable limit of 1000 results](/learn/advanced/known_limitations.md#maximum-number-of-results-per-search). If you want to scrape your database, use the [get documents endpoint](/reference/api/documents.md#get-documents) instead.
```

Search for documents matching a specific query in the given index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.

:::note

> This endpoint has a [non-customizable limit of 1000 results](/learn/advanced/known_limitations.md#maximum-number-of-results-per-search). If you want to scrap your database, you can use the [get documents endpoint](/reference/api/documents.md#get-documents) instead.

Reviewer: ```suggestion
This endpoint has a [non-customizable limit of 1000 results](/learn/advanced/known_limitations.md#maximum-number-of-results-per-search). If you want to scrape your database, you can use the [get documents endpoint](/reference/api/documents.md#get-documents) instead.
```
